packages/runtime/api.rest

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
I also use the `.rest` extension but I'm curious why you're pushing this into source control?

Reply:
Because those are common tasks we do when working with this piece of code, and it gives a good pointer to other devs on how to use the api without digging through the code.

I think every package should have one with common tasks (when they have an api).

For example, on the .rest file in botpress, you can log on, get the list of bots, test converse, etc. without using the UI